.editorconfig

```diff
@@ -8,6 +8,8 @@ root = true
 indent_style = tab
 end_of_line = lf
 charset = utf-8
+indent_style = tabs
```

Comment:
think we had `tabs` confused with `tab` the other day.

Should be `tab` (like line 8), says https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#indent_style